docs/platform/concepts/application-users.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Application users
+limited: true
+---
+
+An application user is a type of user that provides programmatic access to the Aiven platform and services through the [Aiven API](/docs/tools/api.md), [CLI](/docs/tools/), [Terraform](/docs/tools/terraform.md), and [Kubernetes](/docs/tools/kubernetes.md). They're intended for non-human users that need to access Aiven.
+
+:::info
+You must be a [super admin](/docs/platform/howto/make-super-admin) to access this feature.
+:::
+
+## Application user permissions
+
+You [create and manage application users](/docs/platform/howto/manage-application-users)
+at the organization level. Unlike organization users, application users can't log in to
+the Aiven Console. However, application users are granted access to to projects
+and services in the same way as organization users by adding them to
+[projects](/docs/platform/howto/add-project-members) and assigning them a role.
+
+## Security best practices
+
+Because application users can have the same level of access to projects and services it's
+important to secure these accounts and their authentication tokens to avoid abuse. The
+following are some suggested best practices for using Aiven application users.
```

Comment:
```suggestion
Application users can have the same level of access to projects and services, and therefore, it's important to secure these accounts and their authentication tokens to avoid abuse. The following are some suggested best practices for using Aiven application users.
```

Reply:
Trying to avoid starting sentences with because (although grammatically correct, it may be ok in informal writing)